CDCgov/prime-reportstream
# Automate Okta login when the credentials are available

This PR adds automates the Access Token fetching process so that Staging tests can be fully automated. It also includes commands to force refresh and do the old manual method.

Test Steps:
1. Go to `pdhstaging-functionapp` and the Configuration section of it
2. Get the `OKTA_authKey` value from `Application settings` and `export` it on your terminal
3. Run `./prime login --env staging` on the same terminal and confirm there is a log message starting with `Login valid until`
4. Repeat the previous step, and verify the log has a message starting with `Has a valid token until`
5. Run `./prime login --env staging --force` on the same terminal and confirm there once again is a log message starting with `Login valid until`
6. Run `./prime login --env staging --force --manual`, login to Okta via the browser, and verify that in the terminal there once again is a log message starting with `Login valid until`

## Changes
- Add automatic login to `OktaCommand` via the `Client Credentials` Api
- Replace hardcoded Api values from `OktaCommand` with Environmental Variable loading
- Add `manual` and `refresh` options for `OktaCommand`

## Linked Issues
- #10422
- #10964
